doc/userguide/upgrade.rst

```diff
@@ -179,6 +179,12 @@ Removals
   if this behavior is still required. See :ref:`multiple-eve-instances`.
 - Unified2 has been removed. See :ref:`unified2-removed`.
 
+Performance
+~~~~~~~~~~~
+- In YAML files w/o a `flow-timeouts.tcp.closed` setting, the default went from 0 to 10 seconds.
+  This may lead to higher than expected TCP memory use:
+  https://redmine.openinfosecfoundation.org/issues/6552
```

Comment:
Would it make sense to mention the commit that led to the change, mentioned by Thomas? https://github.com/OISF/suricata/commit/c50ef8cc2114ab8d

Reply:
IMO, tickets, not commits belong in documentation. You should then be able to pivot to the commits.